riscv-page-tables/src/page_table.rs

```diff
@@ -518,6 +518,38 @@ impl<T: PagingMode> PageTableInner<T> {
         entry
     }
 
+    /// Creates a translation for `vaddr` to `paddr` with the given permissions.
+    ///Like map_4k_leaf, but with an added page size parameter.
+    ///
+    /// # Safety
+    ///
+    /// The caller must guarantee that `paddr` references a page uniquely owned by the root
+    /// `GuestStagePageTable`.
+
+    unsafe fn map_general_leaf(
```

Comment:
Is there a reason to keep `map_4k_leaf`?

Reply:
gone